src/setup-beam.js

```diff
@@ -254,7 +254,12 @@ async function getOTPVersions(osVersion) {
   let otpVersionsListings
   let originListing
   if (process.platform === 'linux') {
-    originListing = `/builds/otp/${osVersion}/builds.txt`
+    let osArchitecture = getRunnerOSArchitecture()
+    if (osArchitecture.length > 0) {
```

Comment:
We're comparing what seems to be an array to an actual string. Maybe we could have the called function return `undefined` (which it should by default) and compare to `!= undefined`, instead of `.length > 0`?